components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java

```diff
@@ -727,11 +727,27 @@ public APIDefinitionValidationResponse validateAPIDefinition(String apiDefinitio
         } else {
             OpenAPI openAPI = parseAttemptForV3.getOpenAPI();
             io.swagger.v3.oas.models.info.Info info = openAPI.getInfo();
+            List<String> endpoints;
+            String endpointNew = "";
+            if (openAPI.getServers() == null || openAPI.getServers().isEmpty()) {
+                endpoints = null;
+            } else {
+                endpoints = openAPI.getServers().stream().map(url -> url.getUrl()).collect(Collectors.toList());
+                for (String endpoint : endpoints) {
+                    if (endpoint.startsWith("/")) {
+                        if (host.equals("")) {
```

Comment:
When using equals use the convention "String".equals(variable) this will avoid null pointer exceptions in case the variable is null.

Reply:
Fixed with 70996ea3b73a5d43824d1e07752ecc6944fee08c. Used `StringUtils.isEmpty()`.